bedrock-devnet/devnet/__init__.py

```diff
@@ -137,8 +137,9 @@ def devnet_l1_genesis(paths):
     init_devnet_l1_deploy_config(paths)
 
     fqn = 'scripts/Deploy.s.sol:Deploy'
+    # Use foundry pre-funded account #1 for the deployer
     run_command([
-        'forge', 'script', '--chain-id', '900', fqn, "--sig", "runWithStateDump()"
+        'forge', 'script', '--chain-id', '900', fqn, "--sig", "runWithStateDump()", "--private-key", "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
Why exactly do we need this?

Reply:
Here's why. Correct me if something's wrong :)

The existing devnet init script does not specify the sender of the deploy script, so it uses the foundry's default sender([`0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38`](https://book.getfoundry.sh/reference/config/testing?highlight=0x00a329c0648769a73afac7f9381e08fb43dbea72#general)). Then the script deploys an OP chain to its embedded network and transfers all ownership of contracts to the safe contract which is owned by the deployer address.

But the problem is the Asterisc deploy script needs the owner permission of `DisputeGameFactory`, to set the new game impl, which means we need to run the Asterisc deploy script with the same account that is used for devnet deployment. However, it seems impossible to use foundry's default account for transactions to the external network(The devnet L1 network is an external network to the Asterisc deployment script.).

So I just changed the devnet deployer account to the one of pre-funded accounts of foundry, whose private key is publicly known(`test test test test test test test test test test test junk`). I think it's ok because it's only used for the local devnet, and it also can help some tests which require ownership permission.

Another workaround is adding the pre-funded account as the another signer of the safe contract in the deploy script. However, we must carefully modify the deployment script to ensure that it does not affect any prod network deployment.

Reply:
To add more context, foundry's default sender(https://book.getfoundry.sh/reference/config/testing?highlight=0x00a329c0648769a73afac7f9381e08fb43dbea72#general)'s private key is [unknown](https://github.com/foundry-rs/foundry/blob/0026488754512acf0fd902a0d2c90cf8a09367b0/crates/evm/core/src/constants.rs#L19).

Reply:
@clabby Are we good to resolve this?